CommunityFridgeMapApi/functions/fridges/v1/app.py

```diff
@@ -22,14 +22,16 @@ def lambda_handler(event: dict, ddbclient: "botocore.client.DynamoDB") -> dict:
         to the event httpMethod
         """
         httpMethod = event.get("httpMethod", None)
-        path_parameters = event["pathParameters"] or {}
-        fridge_id = path_parameters.get("fridge_id")
+        pathParameters = event["pathParameters"] or {}
+        queryStringParameters = event["queryStringParameters"] or {}
+        tag = queryStringParameters.get("tag", None)
+        fridge_id = pathParameters.get("fridge_id", None)
         db_response = None
         if httpMethod == "GET":
             if fridge_id:
                 db_response = Fridge(db_client=ddbclient).get_item(fridge_id)
             else:
-                raise ValueError(f"Query Fridges API Not Yet Developed")
+                db_response = Fridge(db_client=ddbclient).get_items(tag=tag)
```

Comment:
What happens if `tag` is `None`?

Reply:
if tag is None then `get_items()` will return every fridge in our db https://github.com/CollectiveFocus/CFM_Backend/pull/17/files#diff-9c2aafe3c6ce0facf2c86db6635c7f7c95cd4eeddeceaab949b2da0033f7f44eR390

I made a comment about this here: https://github.com/CollectiveFocus/CFM_Backend/blob/dev/CommunityFridgeMapApi/dependencies/python/db.py#L139

```
        NOTE: This function is probably fine for the fridges in NYC. But as more fridges
        are added to the database, this will be a bottleneck. Ideally we would be querying
        based on proximity. Here is an option for if we ever need to transition:
        https://hometechtime.com/how-to-build-a-dynamodb-geo-database-to-store-and-query-geospatial-data/
```

We'll be tracking latency and see if it gets bad